README.md

```diff
@@ -19,13 +19,17 @@ The cluster definition file enables you to customize your Docker enabled cluster
 
 ## User guides
 
-* [ACS Engine](docs/acsengine.md) - shows you how to build and use the ACS engine to generate custom Docker enabled container clusters
+These guides show how to create your first deployment for each orchestrator:
+* [ACS Engine](docs/acsengine.md) - shows you how to build and use the ACS engine to generate custom container clusters
```

Comment:
This one is general, not specific to an orchestrator so maybe it can go up one line